Bindings/MainWindow.xaml

```diff
@@ -6,6 +6,22 @@
         xmlns:local="clr-namespace:Bindings"
         mc:Ignorable="d"
         Title="MainWindow" Height="450" Width="800">
+    <Window.Resources>
+        <Style x:Key="MathField" TargetType="TextBox">
+            <Setter Property="TextAlignment" Value="Right"/>
+            <Setter Property="VerticalContentAlignment" Value="Center"/>
+        </Style>
+        <Style x:Key="ReadonlyMathField" TargetType="TextBox" BasedOn="{StaticResource MathField}">
+            <Setter Property="IsReadOnly" Value="True"/>
+            <Setter Property="IsReadOnlyCaretVisible" Value="False"/>
+            <Setter Property="Background" Value="{StaticResource {x:Static SystemColors.InactiveSelectionHighlightBrushKey}}"/>
+        </Style>
+        <Style x:Key="MathSymbol" TargetType="Label">
+            <Setter Property="HorizontalContentAlignment" Value="Center"/>
+            <Setter Property="VerticalContentAlignment" Value="Center"/>
+        </Style>
+        <local:VisibilityConverter x:Key="VisibilityConverter"/>
```

Comment:
This is the most common way you see in tutorials and pretty much everywhere.
I prefer adding a static field or property on the converter so that there is no need to use a named resource.
Reason I don't love resources is:
1) Requires the resource to be in scope.
2) Requires a name that must be kept in sync.

Note that what you have is fine and probably the most idiomatic WPF. I'll write an issue about refactoring to static field as a challenge. Merge this and write it as a separate PR.